NeutronStandard/SniffHelpers.php

```diff
@@ -55,7 +55,7 @@ public function getArgumentTypePtr(File $phpcsFile, $stackPtr) {
 
 	public function isReturnValueVoid(File $phpcsFile, $stackPtr) {
 		$tokens = $phpcsFile->getTokens();
-		if ($tokens[$stackPtr]['code'] !== T_RETURN) {
+		if (! in_array( $tokens[$stackPtr]['code'], [T_RETURN, T_YIELD], false)) {
```

Comment:
I wish we had prettier for PHP 😩 . There's an extra space here after the first paren in `in_array(`. If you set your editor to use the phpcs config in this project you'll see a `PSR2.Methods.FunctionCallSignature.SpaceAfterOpenBracket` error here. I should set up this repository to automatically run linting in the CI.